models/ios/logging_global/ios_logging_global.yaml

```diff
@@ -0,0 +1,521 @@
+module: ios_logging_global
+version_added: 2.1.0
+short_description: Manages logging attributes of Cisco IOS network devices
+description: This module manages the logging attributes of Cisco IOS network devices.
+author: Sagar Paul (@KB-perByte)
+options:
+  config:
+    description: A list containing dictionary of logging options
+    type: list
+    elements: dict
+    suboptions:
+      hostname: &host
+        description: IP address of the logging host
+        type: str
+      buffered: &buffered       
```

Comment:
is `&buffered` and `&host` required, as I don't see them being used in the model

Reply:
removed extra substitutions @justjais Thanks